Projects/Embark/Sources/Viewables/Actions/AddressAutocompleteAction/AddressInput.swift

```diff
@@ -0,0 +1,66 @@
+import Flow
+import Foundation
+import Hero
+import Presentation
+import UIKit
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct AddressInput {
+	let placeholder: String
+	let textSignal = ReadWriteSignal<String>("")
+	private let setTextSignal = ReadWriteSignal<String>("")
+	let setIsFirstResponderSignal = ReadWriteSignal<Bool>(true)
+	let shouldReturn = Delegate<String, Bool>()
+	let didEditSignal = ReadWriteSignal<Void>(())
+
+	var text: String {
```

Comment:
this is a bit confusing to me. Whats the point of having two `textSignals` here?